docs/analytics-gtm/pii-remover.md

```diff
@@ -0,0 +1,103 @@
+# Stripping Personally Identifiable Information (PII)
+
+A module exists for stripping out PII from values. It can accept strings, query parameters, arrays or objects. If the value is an array or object, the child values are looped through and then treated as strings.
+
+99% of the code has been taken from the legacy universal analytics PII redaction code.
+
+## What is redacted
+
+The values that are redacted by default are:
+- Email addresses - `email@gov.uk` becomes `[email]`
+- Reset password tokens become `reset_password_token=[reset_password_token]`
+- Unlock tokens become `unlock_token=[unlock_token]`
+- State values become `state=[state]`
+
+Additional values that can be redacted are:
+- Dates - `2022-02-22` becomes `[date]`
+- Postcodes - `sw1a 1aa` becomes `[postcode]`
+
+
+These aren't enabled by default due to the higher likelihood of producing false positives.
+
+Query string parameters are redacted if there is presence of a `meta[name="govuk:static-analytics:strip-query-string-parameters"]` meta tag, with comma separated query string parameters keys in its `content` attribute. The usage of this query string PII removal in production is unknown at this time.
+
+
+### Enabling additional redaction
+
+In order to enable additional redaction you can:
+
+#### Add meta tags to your application via the `Meta Tags` publishing component:
```

Comment:
Not sure this is a heading, it feels like it belongs with the previous sentence, i.e. 

```
In order to enable additional redaction you can add meta tags to your application via the `Meta Tags` publishing component.
```